cli/azd/pkg/tools/dotnet/dotnet.go

```diff
@@ -192,6 +194,31 @@ func (cli *dotNetCli) PublishContainer(
 	return nil
 }
 
+func (cli *dotNetCli) GetTargetPort(
+	ctx context.Context, project string,
+) (int, error) {
+	runArgs := newDotNetRunArgs("publish", "--getProperty:GeneratedContainerConfiguration", project)
+	result, err := cli.commandRunner.Run(ctx, runArgs)
+	if err != nil {
+		return 0, fmt.Errorf("failed to get target port at project '%s': %w", project, err)
+	}
+
+	// Filter the results to only use numbers - we need to reconsider this strategy
+	targetPointString := regexp.MustCompile(`[^0-9]+`).ReplaceAllString(result.Stdout, "")
+
+	// Set default target port 8080
+	if targetPointString == "" {
+		return 8080, nil
+	}
```

Comment:
This strategy is not solid.

Since the `result.Stdout` is a json, what we do is to create an structure with the share of that json.

For example, for a json like:

```
{
  "config": {
    "ExposedPorts": {
      "8080/tcp": {}
    },
    "Labels": {
    }
}
```

You would create:
```
type responseModelExpPorts struct {
   ExposedPorts string `json:ExposedPorts`
}

type responseModel struct {
    config responseModelExpPorts
}
```

Then, you would use json.Marshall/Unmarshal to parse the response bytes into the `responseModel` or to serialize the model back to bytes[]